snap/info.go

```diff
@@ -541,12 +541,13 @@ var _ PlaceInfo = (*Info)(nil)
 type PlugInfo struct {
 	Snap *Info
 
-	Name      string
-	Interface string
-	Attrs     map[string]interface{}
-	Label     string
-	Apps      map[string]*AppInfo
-	Hooks     map[string]*HookInfo
+	Name       string
+	Interface  string
+	Attrs      map[string]interface{}
+	Label      string
+	Apps       map[string]*AppInfo
+	Hooks      map[string]*HookInfo
+	SnapGlobal bool
```

Comment:
Should this be public? It feels like an implementation detail.

In addition can you please move it one line below so that it doesn't influence the block of code above it.